test/maxrows.test.js

```diff
@@ -4,25 +4,27 @@
 // License text available at https://opensource.org/licenses/Artistic-2.0
 
 var should = require('should');
+var db, Thing;
```

Comment:
Nope, this is also reported by eslint, which means you could not use a variable without declaring it.